app/controllers/application_controller.rb

```diff
@@ -753,7 +753,7 @@ def get_reports_menu(group = current_group, tree_type = "reports", mode = "menu"
     user = current_user
     @sb[:grp_title] = reports_group_title
     @data = []
-    if (!group.settings || !group.settings[:report_menus] || group.settings[:report_menus].blank?) || mode == "default"
+    if (!group.settings || group.settings[:report_menus].blank?) || mode == "default"
```

Comment:
Can you remove the parens as well as they are now redundant?  `(A || B) || C == A || B || C`